app/components/layout/Navigation/index.js

```diff
@@ -292,7 +312,9 @@ class Navigation extends React.Component {
 
             {this.renderMyBuilds()}
             <NavigationButton className="py0 xs-hide sm-hide" href={`/docs`}>Documentation</NavigationButton>
-            <NavigationButton className="py0 xs-hide sm-hide" href="mailto:support@buildkite.com">Support</NavigationButton>
+            <NavigationButton className="py0 xs-hide sm-hide" onClick={this.handleSupportClick}>Support</NavigationButton>
```

Comment:
This is missing from the dropdown (mobile) version! Speaking of, how does it look on mobile?